stable/insights-agent/templates/prometheus-metrics/rbac.yaml

```diff
@@ -5,6 +5,12 @@ metadata:
   name: {{ include "insights-agent.fullname" . }}-prometheus-metrics
   labels:
     app: insights-agent
+  {{- if  (index $.Values "prometheus-metrics" "serviceAccount") }}
```

Comment:
can we cobine these?
```
 {{- if  (index $.Values "prometheus-metrics" "serviceAccount") }}
 {{- if  (index $.Values "prometheus-metrics" "serviceAccount" "annotations") }}
```
or not repeat `"prometheus-metrics" "serviceAccount"` which where tested already?

Reply:
without the first if it throws nil pointer

Reply:
and depending on helm version combining could fail